modules/@apostrophecms/seo-fields-global/index.js

```diff
@@ -42,5 +72,38 @@ module.exports = {
         group
       }
       : null;
+  },
+  apiRoutes(self) {
+    return {
+      get: {
+        '/robots.txt': async (req) => {
+          const criteria = {
+            type: '@apostrophecms/global'
+          };
+          try {
+            const globalDoc = await self.apos.doc.find(req, criteria).toObject();
+            let robotsTxtContent = '';
+            switch (globalDoc.robotsTxtSelection) {
+              case 'allow':
+                robotsTxtContent = 'User-agent: *\nDisallow: ';
```

Comment:
Add a trailing newline, because most `robots.txt` files in the wild will have them, and we want to be as normal as possible just in case some bot is picky

Reply:
Added.